### ISSUE TITLE
finetuning

### ISSUE THREAD

**Issue by AminUCo** [jolibrain/deepdetect#437]
(A Markdown syntax reminder is available here: https://guides.github.com/features/mastering-markdown/)

Before creating a new issue, please make sure that:
- [ ] the problem isn't already identified in the in the [FAQ page](http://www.deepdetect.com/overview/faq/ "DeepDetect FAQ page"),
- [ ] your problem isn't listed in an existing issue ([open](https://github.com/beniz/deepdetect/issues) or [closed](https://github.com/beniz/deepdetect/issues?q=is%3Aissue+is%3Aclosed) one),
- [x] [all prerequisite tools](http://www.deepdetect.com/overview/installing/ "DeepDetect installation page") and [dependencies](https://github.com/beniz/deepdetect#user-content-dependencies "DeepDetect README page on Github") are installed.

If Ok, please give as many details as possible to help us solve the problem more efficiently.
#### Configuration
- Version of DeepDetect:
    - [ ] Docker
- Commit (shown by the server when starting):

#### Your question / the problem you're facing:
I want to finetune a pretrained  alexnet  but I dont want to change three first layers ,the problem is i dont know how . i have searched through the entire deepdetect tutorials  and did not find the answer . 
my question is : is it possible to keep some layers unchanged in the fine tuning process ? 
and if the answer is yes i want to know how 

#### Error message (if any) / steps to reproduce the problem:
- [ ] list of API calls:

- [ ] Server log output:

**Comment by beniz** [comment]
Hi, for this you have to edit the `lr_mult` by hand and set them to 0 for every layer you don't want to train. Then create dd service without specifying any `template` to prevent your changes to be written over.

FYI we have never really observed any benefit from finetuning while freezing some layers for the kind of tasks supported by DD. There are some known cases we have experienced, such as image captioning and some NLP tasks. Most of the time finetuning the whole network with a small learning rate yields better results (for us).

**Comment by AminUCo** [comment]
thank you
